.github/workflows/wheels-release.yml

```diff
@@ -0,0 +1,51 @@
+name: build-many-linux-wheels
+
+on:
+  pull_request:
+    branches:
+      - 'main'
+
+jobs:
+  ml-wheels:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Setup Python 3.8
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+
+      - name: upgrade pip & install numpy
+        run: |
+          python -m pip install --upgrade pip
+          pip install numpy==1.20
+
+      - name: Build source dist
+        run: |
+          python setup.py sdist
+
+      - name: Build manylinux Python wheels
+        uses: RalfG/python-wheels-manylinux-build@v0.4.2-manylinux2014_x86_64
+        with:
+          python-versions: 'cp37-cp37m cp38-cp38 cp39-cp39 cp310-cp310'
+          build-requirements: 'cython numpy scipy libsvmdata scikit-learn xarray download'
```

Comment:
Did you check if this will reinstall another version of numpy inside the recipe ?

Reply:
No, I didn't

To make sure that we have the desired numpy version, we can specific it
``build-requirements: 'cython numpy==1.20 scipy libsvmdata scikit-learn xarray download'``

It s possible to do so according to the [gh action doc](https://github.com/RalfG/python-wheels-manylinux-build#:~:text=cp36m%20cp37%2Dcp37m%27-,build%2Drequirements,-Python%20(pip)%20packages)

Reply:
sounds good !
Do you confirm that it's just a build requirement, but that it will not pin numpy for users of celer ?

Reply:
Not totally sure